### PR TITLE
[Chore] Update Market Wars copy

### PR DESCRIPTION
**Pull request by BrettCleary**
# Summary

Updated copy

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -590,15 +590,15 @@
...
                 "subtitle": "Download World of Warcraft 3.3.5a",
-                "title": "Step 1",
-                "webtorrent": "Torrent (Magnet Link)"
+                "title": "Step 1"
             },
             "step2": {
-                "description": "Select the downloaded ChromieCraft_3.3.5a.zip file.",
-                "subtitle": "Select ChromieCraft_3.3.5a.zip File",
+                "description1": "Select the downloaded 3.3.5a zip file.",
+                "subtitle1": "Select the downloaded 3.3.5a zip file",
```
Comment (BrettCleary) [inline public/locales/en/translation.json:601]: added the 1 to the key so that existing translations in other languages won't be used

**Review by jiyuu-jin — APPROVED** [review]

**Review by jiyuu-jin — APPROVED** [review]
LGTM

**Review by flavioislima — APPROVED** [review]
lgtm

**Review by nyghtstalker — APPROVED** [review]
Confirmed that the updated text on the modal is accurate. Also, confirmed that the Game7 popup modal no longer triggers when users go to the quests page.

image

**Pull request merged**